wicket/src/rack_setup/config_toml.rs

```diff
@@ -0,0 +1,296 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Support for the TOML file we give to and accept from clients for setting
+//! (most of) the rack setup configuration.
+
+use serde::Serialize;
+use std::borrow::Cow;
```

Comment:
I think the "download/upload TOML" is a temporary placeholder and ultimately we want to enter the data directly in wicket, right? So I think the wicket <-> wicketd interface should look like what we want in the long run, approximately, and the placeholder stuff can live in wicket where it will naturally be replaced in time.